src/main/java/TaskList.java

```diff
@@ -0,0 +1,103 @@
+import java.util.ArrayList;
+
+public class TaskList {
+
+  public ArrayList<Listing> list;
+
+  /**
+   * Takes in a new Arraylist<Listing> and initialises list
+   */
+  public TaskList(ArrayList<Listing> l) {
+    this.list = l;
+  }
+
+  /**
+   * Prints the toString() value of each listing in the list
+   */
+  protected void printReturns() {
+    System.out.println("    ____________________________________________________________");
+    System.out.println("     Here are the tasks in your list:");
+    for (int i = 0; i < list.size(); i++) {
+      String s = "     " + (i + 1) + "." + list.get(i).toString();
+      System.out.println(s);
+    }
+    System.out.println("    ____________________________________________________________");
+  }
+
+  /**
+   * Modifies the done boolean in the listing objects inside the list. Print the correct message
+   * through printer * and updates storage through the storage input * @param num used to find the
```

Comment:
I think your @param has to be on different lines like your other methods?

Reply:
Done